src/clarity-angular/popover/signpost/signpost.ts

```diff
@@ -211,33 +230,32 @@ export class Signpost {
         if ( position && (signpostPositions.indexOf(position) > -1) ) {
             this._signpostPostion = position;
         } else {
-            console.log("Setting default");
             this._signpostPostion = "right-middle";
         }
         // // set the popover values based on menu position
         switch ( this._signpostPostion ) {
             case ("top-left"):
-                this.anchorPoint = Point.TOP_LEFT;
+                this.anchorPoint = Point.TOP_CENTER;
                 this.popoverPoint = Point.BOTTOM_RIGHT;
-                this.signpostOptions.offsetY = -8;
-                this.signpostOptions.offsetX = 10;
+                this.signpostOptions.offsetY = -10;
```

Comment:
It seems that storing these offset values in an enum might be easier to maintain over time. Would you agree?

This isn't something I would block the PR on. But could you add a `TODO` for it? It would make a great `contributions-welcome` task/chore...